lib/srv/desktop/rdp/rdpclient/librdprs.h

```diff
@@ -52,6 +52,30 @@ typedef enum CGOPointerWheel {
   PointerWheelHorizontal,
 } CGOPointerWheel;
 
+typedef enum FileType {
+  File = 0,
+  Directory = 1,
+} FileType;
+
+typedef enum TdpErrCode {
+  /**
+   * nil (no error, operation succeeded)
+   */
+  Nil = 0,
```

Comment:
Agreed, but it will be a big pain to do here so I will do it once the subsequent branches are merged.

https://github.com/gravitational/teleport/issues/13525